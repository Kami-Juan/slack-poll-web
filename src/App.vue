<template>
  <div id="app">
    <Poll />
  </div>
</template>

<script>
import Poll from './components/Poll.vue'
import 'bootstrap/dist/css/bootstrap.css'
import 'bootstrap-vue/dist/bootstrap-vue.css'

export default {
  name: 'app',
  components: {
    Poll
  }
}
</script>

<style>
@font-face {
  font-family: LatoBold;
  src: url(/fonts/Lato-Bold.ttf) format('truetype');
}

@font-face {
  font-family: Lato;
  src: url(/fonts/Lato-Regular.ttf) format('truetype');
}

* {
  background-color: #1B1D21;
  color: white;
  font-family: Lato;
}

#app {
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  margin-top: 60px;
}

:root {
  --main-font-secondary-color: rgba(209, 210, 211, 0.5);
  --main-font-primary-color: rgba(209, 210, 211, 1);
  --main-border-color: rgba(216, 216, 216, 0.5);
}
</style>
