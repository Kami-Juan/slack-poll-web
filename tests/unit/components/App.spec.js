import {mount} from '@vue/test-utils'
import App from '../../../src/App'

describe('test App.js', () => {
  let cmp

  beforeEach(() => {
    cmp = mount(App, {
      stubs: {
        Poll: '<div id="stub-poll"></div>'
      }
    })
  })

  it('should be render', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
