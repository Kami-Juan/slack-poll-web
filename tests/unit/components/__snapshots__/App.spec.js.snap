// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test App.js should be render 1`] = `
<div
  id="app"
>
  <div
    id="stub-poll"
  />
</div>
`;
